src/components/CardWithArrow/CardWithArrow.module.css

```diff
@@ -26,23 +29,30 @@
   transform: rotate(-45deg);
 }
 
-.subtitle {
-  margin-bottom: 0.5rem;
+.subtitle, .icon  {
+  margin-bottom: 1rem;
 }
 
-.title {
-  font-weight: 700;
-  color: var(--neural-950-color);
-  max-width: 13.5625rem;
-  transition: all 0.2s ease-in-out;
+.link_text {
```

Comment:
are you sure this is not already defined in the Global CSS? all font sizes should be there (figma should tell you if it uses a predefined font)